### PR TITLE
Fixes to allow Windows development

### PR DESCRIPTION
**Pull request by smikula**
These are a handful of fixes to get everything to play nice in my Windows environment.  I think I fixed it in a way that should work in linux and elsewhere, but I'm only able to test in Windows.

### CODE REVIEW

File: wsgi.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 if settings.DEBUG:
 	import subprocess
 
-	webpack_bin = os.path.join(settings.BASE_DIR, "node_modules/.bin/webpack")
+	webpack_bin = os.path.join(settings.BASE_DIR, "node_modules/.bin/webpack.cmd")
```
Comment (jleclanche) [inline wsgi.py]: That will break Linux.

Comment (smikula) [inline wsgi.py]: I just pushed a fix to only spawn the .cmd on windows.

**Comment by smikula** [comment]
Okay, I fixed up the entire branch to get rid the stale commit.  Should be in a good state now.

**Comment by smikula** [comment]
Squashed my wsgi.py fixes together

File: webpack.config.js [hunk 2]
```diff
@@ -1,16 +1,26 @@
+"use strict";
+
 const path = require("path");
 const webpack = require("webpack");
 const BundleTracker = require("webpack-bundle-tracker");
 const fs = require("fs");
-const {spawnSync} = require("child_process");
-
+const spawnSync = require("child_process").spawnSync;
```
Comment (jleclanche) [inline webpack.config.js]: Why this change?

Comment (smikula) [inline webpack.config.js]: This was necessary for it to run in Node v5.12.0.  (That's what I'm using, and I don't have a lot of flexibility to upgrade since I'm pegged to that version for work things.)  Seems like a win to make one minor change and be able to support a wider range of dev environments.

Comment (jleclanche) [inline webpack.config.js]: We use https://github.com/ekalinin/nodeenv, i dont know how well that works on Windows though.

**Comment by jleclanche** [comment]
I landed a separate fix for the last issue in ec28693.

File: webpack.config.js (hunk 2 above)
Comment (beheh) [inline webpack.config.js]: @smikula Since this went unfixed: I'm not really sure I generally agree with the motivation, but for now I'll land d874d0092fde292444b2c8e2a1fb442514121d11 which removes the destructuring in webpack.config.js. I can't promise we won't introduce Node v6 features here in future though. I can only also recommend using `nodeenv` or a system where node can be updated regularly (I've personally never run into issues with Node's backwards compatibility).

BTW: The destruring assignment specifically should work even in Node v5 using `node --harmony_destructuring`.
